pkg/util/scrubber/scrubber.go

```diff
@@ -40,6 +42,17 @@ type Replacer struct {
 	// ReplFunc, if set, is called with the matched bytes (see regexp#Regexp.ReplaceAllFunc). Only
 	// one of Repl and ReplFunc should be set.
 	ReplFunc func(b []byte) []byte
+
+	// ImplementVersion is the version of the implementation of the replacer.
+	ImplementVersion *version.Version
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Do we care about the version is was implemented or rather the version it was last changed ?
I'm thinking the regex is fixed at some point then it should be applied to every older version

Reply:
I will change the name, it should be lastUpdated or something like that,

It should be the version when the regex got updated for the last time